test/sequential/bootstrap.claimAndActivate.test.ts

```diff
@@ -0,0 +1,278 @@
+/* eslint-disable jest/no-conditional-expect */
+/*
+ *
+ * @group bootstrap
+ * @group sequential
+ */
+import { getApi, initApi } from "../../utils/api";
+import {
+  scheduleBootstrap,
+  provisionBootstrap,
+  provisionVestedBootstrap,
+  finalizeBootstrap,
+  getLiquidityAssetId,
+  claimAndActivateBootstrap,
+  promotePool,
+  vestingTransfer,
+} from "../../utils/tx";
+import { EventResult, ExtrinsicResult } from "../../utils/eventListeners";
+import { Keyring } from "@polkadot/api";
+import { User } from "../../utils/User";
+import {
+  getEnvironmentRequiredVars,
+  waitForNBlocks,
+  getBlockNumber,
+} from "../../utils/utils";
+import { getEventResultFromMangataTx } from "../../utils/txHandler";
+import { MGA_ASSET_ID } from "../../utils/Constants";
+import { BN, toBN } from "@mangata-finance/sdk";
+import { Assets } from "../../utils/Assets";
+
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+jest.spyOn(console, "error").mockImplementation(jest.fn());
+jest.setTimeout(3500000);
+process.env.NODE_ENV = "test";
+
+let testUser1: User;
+let testUser2: User;
+let sudo: User;
+let keyring: Keyring;
+let bootstrapPhase: any;
+let bootstrapCurrency: any;
+let bootstrapPool: any;
+let bootstrapUser1Liquidity: any;
+let bootstrapUser2Liquidity: any;
+let bootstrapExpectedUserLiquidity: BN;
+let eventResponse: EventResult;
+
+const { sudo: sudoUserName } = getEnvironmentRequiredVars();
+const waitingPeriod = 15;
+const bootstrapPeriod = 30;
+const bootstrapAmount = new BN(10000000000);
+
+async function bootstrapRunning(
+  promotedPool: boolean,
+  vestedProvision: boolean
+) {
+  const api = getApi();
+  if (vestedProvision === true) {
+    const bootstrapBlockNumber = (await getBlockNumber()) + 10;
+    await sudo.addMGATokens(sudo);
+    const vestingUser = await vestingTransfer(
+      sudo,
+      MGA_ASSET_ID,
+      sudo,
+      testUser1,
+      bootstrapBlockNumber
+    );
+    eventResponse = getEventResultFromMangataTx(vestingUser);
+    expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
+  }
+  await testUser1.addMGATokens(sudo);
+  await sudo.mint(bootstrapCurrency, testUser1, toBN("1", 20));
+  await testUser2.addMGATokens(sudo);
+  await sudo.mint(bootstrapCurrency, testUser2, toBN("1", 20));
```

Comment:
we can save some time using this parallelization  @marian created. 
https://github.com/mangata-finance/mangata-e2e/blob/main/test/sequential/xyk-pallet.API.fees.v2.test.ts#L41
It's about minting mangatas and this is a sequential test, so maybe we can parallelize this !